### PR TITLE
Split the cache table from the record table

### LINKED ISSUE #26: Split cached records table from the main records table
Currently, the cached records are mixed in with the static records.

This will cause some issues when displaying records from the database so I would like another table specifically for the cache.

### PR DESCRIPTION
**Pull request by jmeaster30**
- Added a new table for cached records
- Hooked up new table to lookup in the resolver and made it so responses are cached and cleared properly
- fixed a renaming bug I introduced in the TUI changes

Resolves #26

### CODE REVIEW

**Comment by jmeaster30** [comment]
To-do list
- [ ] Finish getting cached records from db for TUI
- [ ] Maybe make a From derive macro that implements From<MyType>, From<&MyType>, FromIterator<&MyType>, etc. to a particular other type
- [ ] switch between views

**Comment by jmeaster30** [comment]
Resolves #22

**Comment by jmeaster30** [comment]
Resolves #8

File: Cargo.toml [hunk 1]
```diff
@@ -14,13 +14,14 @@ rand = "0.8.5"
...
+simple-macros = { path = "simple-macros" }
 tabled = "0.17.0"
 yaml-rust = "0.4"
 
 [features]
-default = ["log_info", "log_warn", "log_error"]
-log_info = []
-log_debug = []
-log_warn = []
+default = ["log_info"]
```
Comment (witchofthewires) [inline Cargo.toml:22]: Is this configured the way you want? You don't want "log_info = ['log_info']"?

**Review by witchofthewires — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (jmeaster30) [reply Cargo.toml:22]: Yeah these are basically feature flags and not really related to the macros themselves.

In the "log_info" macro, we check if the "log_info" feature flag is set and if not, we dont run the code.

**Pull request merged**